os_migrate/roles/conversion_host/tasks/conv_hosts_inventory.yml

```diff
@@ -27,6 +27,7 @@
       os_migrate_conversion_client_cert: "{{ os_migrate_dst_client_cert|default(omit) }}"
       os_migrate_conversion_client_key: "{{ os_migrate_dst_client_key|default(omit) }}"
 
+# TODO-storage do we need to handle this at all, given if data_copy is false then we dont deploy a src CH ?
```

Comment:
@jistr want to confirm this as well, do we want to search for conversion host at all?